test_windprofile.ipynb

```diff
@@ -0,0 +1,270 @@
+{
+ "cells": [
+  {
```

Comment:
@alicebarthel, I don't think this file should have been committed at the top level.